new-module/README.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Angle _New_ Module
+---
+
+# Angle _New_ Module
+
+Angle Borrow module goal is to help expand the Angle Protocol through another minting mechanism for Angle stablecoins. It is based on a debt mechanism, similar to those used by Maker with DAI, or Abracadbra with MIM. Users can deposit tokens as collateral into the protocol, and borrow agTokens from this deposit depending on specific parameters. 
+
+The main advanatages of Angle Borrow module are: 
+* No need for hedging agents to cover the protocol
+* Easier to add new collaterals
+* Users can keep exposure to their collateral tokens while using agEUR
+
+These advantages will help further expand Angle and its agTokens, both in terms of the stablecoins issued, their volume, and the chains and networks where they can be issued/redeemed. It has been designed to work hand in hand with the current one, and strenghten the Angle Protocol. AgTokens, like agEUR, will be fully interoperable between both modules.
```

Comment:
Let's use present everywhere to avoid changing it when it's live. We can put a hint at the top of this page to say: borrowing module will be live in May